### PR TITLE
Add a table for phase data in tooltips; COUNTRY=cameroon

### PR DESCRIPTION
**Pull request by ericboucher**
This fixes issue #794 

This PR:
- updated the size of the tooltip (increased max height) and changed fonts
- allowed it to be below the point if it fits better on the screen
- allowed users to hide the data title
- creates a table view for Phase Classification data

Screenshot 2023-06-12 at 6 15 20 PM

Screenshot 2023-07-20 at 5 55 22 PM

### CODE REVIEW

**Comment by wadhwamatic** [comment]
@ericboucher - would Marc have some time to give us some design options? The two current use cases are mentioned in https://github.com/WFP-VAM/prism-app/issues/794. There could be a use case for the decision maker view too.

**Comment by ericboucher** [comment]
Screenshot 2023-07-14 at 4 07 58 PM

**Review by jbperidypathtech — APPROVED** [review]
I approve because it is ok for me after last comment accounted.

File: frontend/src/components/MapView/MapTooltip/index.tsx [hunk 1]
```diff
@@ -6,15 +6,86 @@ import {
...
+import { PopupData, tooltipSelector } from 'context/tooltipStateSlice';
 import { isEnglishLanguageSelected, useSafeTranslation } from 'i18n';
+import { TFunction } from 'utils/data-utils';
+import { ClassNameMap } from '@material-ui/styles';
+
+// This function prepares phasePopulationTable for rendering and is specific
+// to the data structure of the phase classification layer.
+// Note - this is a bit hacky for now and will likely need to be revamped if we
+// encounter other complex needs for tooltips.
+const generatePhasePopulationTable = (
```
Comment (JulienPathtech) [inline frontend/src/components/MapView/MapTooltip/index.tsx:21]: We could add `JSX.Element` as return value for this method ? We could also make a separate component file but it depends on the revamp you have in mind

**Pull request merged**